TourMate/TourMate/Models/DomainModels/NewPlan.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NewPlan.swift
+//  TourMate
+//
+//  Created by Keane Chan on 14/3/22.
+//
+
+import Foundation
+
+struct NewPlan {
+    let id: String
+    let tripId: String
+    var planType: PlanType
+    var name: String
+    var startDate: Date
+    var endDate: Date
+    var timeZone: TimeZone
+    var imageUrl: String?
+    var status: PlanStatus
+    let creationDate: Date
+    var modificationDate: Date
+
+    // Currently unused
+    // let upVote: Int
+    // let downVote: Int
+    // let comments: [Comments]
+}
```

Comment:
Should `NewPlan` be a protocol? Basically, replacing the `Plan` that we have currently? And is there a need to store 'planType` in the domain modal?

Reply:
Yup I am not too sure whether we need a `planType` as well, but thought I add it in for now in case we do. Will evaluate again in the next PR whether we need it. Thanks